app/static/js/interactive-nav.js

```diff
@@ -0,0 +1,56 @@
+document.addEventListener("DOMContentLoaded", () => {
+    // nav animation on mobile devices
+    const accordionToggle = document.querySelector("[data-toggle='accordion']")
+    const scrollTopButton = document.getElementById("scroll-top-button")
+    const closeAccordionButton = document.querySelector(
+        "[data-toggle='accordionCloseOnly']",
+    )
+    const accordionList = document.getElementById("accordion-list")
+    const navLinkClick = document.querySelectorAll(
+        "[data-toggle='click-nav-link']",
+    )
+
+    if (accordionToggle && accordionList && scrollTopButton) {
+        accordionToggle.addEventListener("click", () => {
+            accordionList.classList.toggle("max-h-screen")
+            scrollTopButton.classList.toggle("translate-x-full")
+            closeAccordionButton.classList.toggle("hidden")
+        })
+    }
+
+    if (closeAccordionButton && accordionList) {
+        closeAccordionButton.addEventListener("click", () => {
+            accordionList.classList.remove("max-h-screen")
+            scrollTopButton.classList.remove("translate-x-full")
+            closeAccordionButton.classList.add("hidden")
+        })
+    }
+    if (accordionList && navLinkClick) {
+        Array.apply(null, navLinkClick).forEach((link) => {
+            link.addEventListener("click", () => {
+                accordionList.classList.remove("max-h-screen")
+                scrollTopButton.classList.remove("translate-x-full")
+                closeAccordionButton.classList.add("hidden")
+            })
+        })
+    }
+
+    // slider for the hero section on desktop
+    const openDesktopNav = document.querySelector(
+        "[data-toggle='open-nav-desktop']",
+    )
+    const closeDesktopNav = document.querySelector(
+        "[data-toggle='close-nav-desktop']",
+    )
+
+    const navSlider = document.getElementById("slider")
+
+    if (openDesktopNav && closeDesktopNav && navSlider) {
+        openDesktopNav.addEventListener("click", () => {
+            navSlider.classList.add("-translate-x-1/2")
```

Comment:
I don’t know what’s supposed to happen in desktop mode, but this doesn’t really open the slider menu when the menu is on the left side (since the slider is kind of hidden in bigger viewports?).

https://github.com/technologiestiftung/innovationserhebung/assets/6429568/a0612e0a-eca5-433d-93f1-9e02fe1b2bf4